frontend/src/components/UserSignUp/ReturnButton.jsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import styled from "styled-components";
+import PropTypes from "prop-types";
+
+export default function ReturnButton({ link }) {
+  const BackArrow = styled.div`
+    background-image: url("WhitePrevButton.png");
+    width: 40%;
+    height: 70%;
+    top: 0%;
+    right: 0%;
+    background-repeat: no-repeat;
+    background-position: center;
+  `;
+
+  const ReturnButtonWrapper = styled.a`
+    background-color: blue;
+    border: none;
+    display: flex;
+    align-items: center;
+    height: 50px;
+    width: 100px;
+    background-size: 100% 50%
+    flex-direction: row;
+    text-decoration: none;
+  `;
+
+  const ReturnText = styled.p`
+    background-color: blue;
+    color: white;
+    font-size: 18px;
+  `;
+
+  return (
+    <ReturnButtonWrapper href={link}>
```

Comment:
FYI we can't use `<a>` tags since we're using React Router. Some code [like this](https://www.codegrepper.com/code-examples/javascript/how+to+go+back+to+the+previous+page+on+button+click+react) should suffice. Sorry for not mentioning this earlier!